src/main/java/at/ac/tuwien/kr/alpha/common/atoms/Atom.java

```diff
@@ -20,13 +47,13 @@
 	 * List of all variables occurring in the Atom that are potentially binding, i.e., variables in positive atoms.
 	 * @return
 	 */
-	List<VariableTerm> getBindingVariables();
+	List<VariableTerm> getBindingVariables(boolean negated);
```

Comment:
I think this `boolean negated` is a bad idea, because the outside now must know which kind of Atom this is and the rest of the refactoring actually tried to move negation outside of `Atom` (which is a good idea!).